drafts/build_a_dao_on_fuel/draft.md

```diff
@@ -0,0 +1,1416 @@
+[note]: <> ("note to self and file editor, there are specific comments left throughout this file, for the sake of understanding the author's reason for using a particular method of approach; please feel free to delete once acknowledged or reply edit with a better approach 🥂")
+
+# How to Build a DAO on the Fuel blockchain.
+
+## Introduction
+
+A Decentralized Autonomous Organization (DAO) is considered an essential blockchain application.
+
+It is a decentralized application that users or organizations use to automate the governance execution of their projects and communities. The automated program is represented by rules encoded in a network of computers that is transparently controlled by the organization members and not influenced by any central authority.
+
+By completing this article, you will learn how to create a Dao contract on the Fuel blockchain. You will also gain knowledge about the following concepts:
+
+1. Writing a smart contract using the sway language.
+2. Understanding how to use low-level code fundamentals in sway `asm`, `call`, etc.
+3. Wallet configuration
+4. How to compile and deploy to the Fuel blockchain (testnet).
+
+## Prerequisites
+
+To follow this tutorial seamlessly, you must have the following checked out.
+
+1. [Installation](https://docs.fuel.network/guides/installation/) of dependencies like `fuel`, `sway` extensions on vscode, `rustc`, `cargo`, `rustup` `cargo-generate`, on your pc.
+2. Node version ^18.18.2 || ^20.0.0.
+3. You should have `wsl` installed if you are using a Windows or Mac PC. You can use the code `wsl --install` to install `wsl`.
+4. Knowledge of FuelVM and other Fuel tools.
```

Comment:
I don't think this one is actually a required pre-req to follow this tutorial. Your tutorial is introducing them to the tools, and knowing the inner workings of the VM won't make this tutorial any easier or simpler to follow.

Reply:
Done
I've made the final requested changes

Reply:
Just confirming, Is this supposed to be removed?